pkgs/by-name/wa/waywall/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  egl-wayland,
+  libspng,
+  libxkbcommon,
+  luajit,
+  mesa_glu,
+  meson,
+  ninja,
+  pkg-config,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  xorg,
+  xwayland,
+}:
+stdenv.mkDerivation {
+  pname = "waywall";
+  version = "0-unstable-2025-02-07";
+
+  src = fetchFromGitHub {
+    owner = "tesselslate";
+    repo = "waywall";
+    rev = "be96e20997c5886af9661d9832b7902aba1e5311";
+    hash = "sha256-77GbBzHjyZuauhl0vlguUS/7jBT4qNjOLYGWBVTPjEY=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wayland-protocols
+    wayland-scanner
+  ];
+
+  buildInputs = [
+    egl-wayland
```

Comment:
Upstream requires `wayland-egl` which is provided by the `wayland` package.

`egl-wayland` is actually NVIDIA's implementation of EGLStreams (see https://github.com/NVIDIA/egl-wayland/)

Please remove this.